Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [CalyxOS/Seedvault](https://hosted.weblate.org/projects/calyxos/seedvault/).


It also includes following components:

* [CalyxOS/Seedvault contacts backup](https://hosted.weblate.org/projects/calyxos/seedvault_contactsbackup/)

* [CalyxOS/Seedvault storage backup](https://hosted.weblate.org/projects/calyxos/seedvault_storage/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/calyxos/seedvault/horizontal-auto.svg)

chirayudesai commented:
Will handle android15 for this later.

lucasmz-dev reviewed (approved):
LGTM

lucasmz-dev commented:
@grote any reason to not get this merged already? This is quite a few months' work + a bunch of new seedvault feats. and I don't see any obvious reason, might as well reduce the labor needed later in case of any issues

grote commented:
this is targeting `android14`, but we are on `android15`, so this needs a bit of work with weblate to straighten this out. @mikeNG knows how.

mikeNG commented:
> this is targeting `android14`, but we are on `android15`, so this needs a bit of work with weblate to straighten this out. @mikeNG knows how.

This PR needs to be merged on `android14` and at the same time we should disable the weblate project until the switch to `android15` happens. These changes will need to be cherry-picked to the `android15` branch manually once and after that, weblate will continue with translations on `android15` branch once we enable the project there again.

chirayudesai commented:
Weblate committed, pushed and locked.

chirayudesai commented:
remote:   https://review.calyxos.org/c/CalyxOS/platform_packages_apps_Seedvault/+/33719 Merge tag '14-5.1' of https://github.com/seedvault-app/seedvault [NEW]
remote:   https://review.calyxos.org/c/CalyxOS/platform_packages_apps_Seedvault/+/33720 Merge tag '14-5.2' of https://github.com/seedvault-app/seedvault [NEW]

chirayudesai commented:
Unfortunately I tagged 14-5.2 before noticing that I hadn't yet merged this.

However, that makes it identical to 15, the only differences between 15-5.2 and 14-5.2 are Android 14 changes, translations are identical.

Will figure out what to do.

weblate committed:
Import translations from Weblate

Co-authored-by: Andi Chandler <andi@gowling.com>
Co-authored-by: Anonymous <noreply@weblate.org>
Co-authored-by: Fjuro <fjuro@alius.cz>
Co-authored-by: Hasáky gum <caro.jf@protonmail.com>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Idcrafter <minefater1@gmail.com>
Co-authored-by: Igor Sorocean <sorocean.igor@gmail.com>
Co-authored-by: Jerguš Fonfer <caro.jf@protonmail.com>
Co-authored-by: Kristian Nordin <kristian.po@proton.me>
Co-authored-by: Languages add-on <noreply-addon-languages@weblate.org>
Co-authored-by: LucasMZ <git@lucasmz.dev>
Co-authored-by: Michael Bestas <mkbestas@gmail.com>
Co-authored-by: Milo Ivir <mail@milotype.de>
Co-authored-by: Oğuz Ersen <oguz@ersen.moe>
Co-authored-by: Renko <kull@tutanota.de>
Co-authored-by: Ricky Tigg <ricky.tigg@gmail.com>
Co-authored-by: Serhii Horichenko <serhii@horichenko.com>
Co-authored-by: Yaron Shahrabani <sh.yaron@gmail.com>
Co-authored-by: abc0922001 <abc0922001@hotmail.com>
Co-authored-by: gallegonovato <fran-carro@hotmail.es>
Co-authored-by: hhs bbsb <bamboo@ik.me>
Co-authored-by: hsfg6 <hsfg6@proton.me>
Co-authored-by: ikya91 <weblate.haq9g@simplelogin.com>
Co-authored-by: lucasmz-dev <git@lucasmz.dev>
Co-authored-by: nautilusx <translate@disroot.org>
Co-authored-by: nobody <info@localcdn.org>
Co-authored-by: pa cs <cs.pa@users.noreply.hosted.weblate.org>
Co-authored-by: rehork <cooky@e.email>
Co-authored-by: sysxda <coderz35@gmail.com>
Co-authored-by: Максим Горпиніч <mgorpinic2005@gmail.com>
Co-authored-by: தமிழ்நேரம் <anishprabu.t@gmail.com>
Co-authored-by: さしみ <kazasiuun@gmail.com>
Co-authored-by: 大王叫我来巡山 <hamburger2048@users.noreply.hosted.weblate.org>
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/ca/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/cs/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/de/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/el/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/en_GB/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/es/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/fi/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/fr/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/he/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/hr/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/ja/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/nl/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/pl/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/pt_BR/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/ro/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/ru/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/sk/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/sv/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/ta/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/tr/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/uk/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/zh_Hans/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault/zh_Hant/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_contactsbackup/en_GB/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_contactsbackup/hr/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_contactsbackup/sk/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_contactsbackup/ta/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_storage/cs/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_storage/de/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_storage/el/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_storage/en_GB/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_storage/es/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_storage/he/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_storage/ja/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_storage/pl/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_storage/pt_BR/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_storage/ro/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_storage/sk/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_storage/sv/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_storage/ta/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_storage/tr/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_storage/uk/
Translate-URL: https://hosted.weblate.org/projects/calyxos/seedvault_storage/zh_Hans/
Translation: CalyxOS/Seedvault
Translation: CalyxOS/Seedvault contacts backup
Translation: CalyxOS/Seedvault storage backup

lucasmz-dev commented:
@mikeNG hey mike, can you also unlock the two other seedvault components? (Local contacts backup and seedvault's storage backup?)
